client/signup/steps/difm-site-picker/styles.scss

```diff
@@ -1,3 +1,22 @@
+body.is-section-signup {
```

Comment:
Is this not redundant? This line of code will not be necessary if every `.signup__step` is contained by `.is-section-signup`.

Reply:
Ah yes there was a competing selector preventing me from changing the color, so I did this to increase specificity. ⚔️